### PR TITLE
4 feature 7 course enrolment course creation - Course class

### PR DESCRIPTION
**Pull request by jltng**
created class skeleton of Course entity (instance attributes not yet properly initialized to respective entities)

### CODE REVIEW

File: src/main/java/Course.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import java.util.ArrayList;
+
+// entity layer
+
+public class Course {
+    /*
```
Comment (yyutiw) [inline src/main/java/Course.java:6]: Use Javadoc comments instead of regular multiline comments for classes and methods

**Review by yyutiw — DISMISSED** [review]
Attributes needed for Task creation + maintenance are present (Course class exists).

**Review by CC-3636 — APPROVED** [review]
Nice job, good documentation

File: src/main/java/Course.java (hunk 1 above)
Comment (jltng) [reply src/main/java/Course.java]: fixed

**Comment by TiareMar** [comment]
Looks good

**Review by TiareMar — APPROVED** [review]

**Review by yyutiw — APPROVED** [review]
Looks good!

**Review by CC-3636 — APPROVED** [review]
Looks good!

**Pull request merged**